mlir/lib/RocmlirTransforms/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Instead of using the LLVM and LLD living next to MLIR, use the COMGR interface
+# to invoke the versions of these tools shipped with ROCm. This is used to
+# avoid packaging a separate copy of the compiler when creating MLIR-based tools
+# targetting ROCm, espcecially those living inside the ROCm distribution itself.
+if (ROCMLIR_ENABLE_COMGR)
+  find_package(amd_comgr REQUIRED PATHS /opt/rocm)
```

Comment:
I suspect this won't work with pytorch's build environment. It'd be good if you can grab their docker and give it a test.